src/test/java/unit/GcsBucketLifecycle.java

```diff
@@ -333,12 +335,23 @@ void remove() throws IOException {
     // `terra resource update gcs-bucket --name=$resourceName --lifecycle=$lifecycle2"
     String lifecycleFilename2 = "empty.json";
     Path lifecycle2 = TestCommand.getPathForTestInput("gcslifecycle/" + lifecycleFilename2);
-    TestCommand.runCommandExpectSuccess(
-        "resource", "update", "gcs-bucket", "--name=" + resourceName, "--lifecycle=" + lifecycle2);
+    var updatedBucket =
```

Comment:
This is the `remove()` test. Please change the `update()` test.

Instead of adding a separate `terra resource update`, I recommend changing the existing `terra resource update` command. This way, the test won't be slower.

Reply:
done. not sure whether my understanding is correct or not